### PR TITLE
removing local.sql as no longer required, amending readme

### PR DESCRIPTION
**Pull request by emmetdev**
The previous way of applying test fixtures to the local development psql database is no longer required because it has been replaced with a database access object approach using the notifications-api. The new test fixtures code can be found at notifications-api/app/commands.py which is called using Make targets in notifications-functional-tests.

https://trello.com/c/togIFJOD/898-remove-the-old-sql-fixtures-from-notifications-functional-tests-and-update-the-readme

### CODE REVIEW

**Review by robinjam — CHANGES_REQUESTED** [review]
Could you also update the "Database fixtures" section further down in the readme?

File: README.md [hunk 1]
```diff
@@ -44,18 +44,12 @@ export FUNCTIONAL_TESTS_LOCAL_API_HOST=http://notify-api.localhost:6011
...
-Populate the local database with fixture data:
+Populate the local database with fixture data using the make target, this will call database fixtures located in notifications-api:
 
 ```shell
-psql notification_api -f db_fixtures/local.sql
-
-# or for dockerised through notifications-local
-
-psql postgresql://notify:notify@localhost:5433/notification_api -f db_fixtures/local.sql
+Make generate-local-dev-db-fixtures
```
Comment (robinjam) [inline README.md:50]: Also `make` needs to be lowercase

Comment (emmetdev) [reply README.md]: Switched to lower case

File: README.md [hunk 2]
```diff
@@ -136,7 +130,7 @@ More groups generally equals better parallelisation (limited by test runner coun
 
 For the functional tests to pass in any environment, they require certain database fixtures to exist.
 
-For your local environment, these fixtures are found in db_fixtures/local.sql. There are [instructions on how to update the local database fixtures](docs/update-local-db-fixtures.md).
+For your local environment, these fixtures are now generated using the custom flask command functional-test-fixtures in notifications-api/app/command.py. The command will create all the database rows required for the funcitonal tests in an idempotent way and output an environment file the functional tests can use to execute against the environment. There are [instructions on how to update the local database fixtures](docs/update-local-db-fixtures.md).
```
Comment (robinjam) [inline README.md:133]:
```suggestion
For your local environment, these fixtures are now generated using the custom flask command functional-test-fixtures (defined in [notifications-api/app/functional_tests_fixtures/__init__.py](https://github.com/alphagov/notifications-api/blob/main/app/functional_tests_fixtures/__init__.py)). The command will create all the database rows required for the funcitonal tests in an idempotent way and output an environment file the functional tests can use to execute against the environment. There are [instructions on how to update the local database fixtures](docs/update-local-db-fixtures.md).
```

File: docs/update-local-db-fixtures.md [hunk 3]
```diff
@@ -8,12 +8,10 @@ Below please find a step-by-step guide for updating db_fixtures/local.sql:
...
-6. Back up your new functional test data with `pg_dump --format=p --data-only notification_api > temp_db_setup_fixtures.sql`
-7. Compare the new fixtures file with the existing one and remove any extra rows that people won't want to re-add to their database. For example, Notify service templates, any notifications you might have sent while making the changes, etc.
-8. (Optional) revert your local to its previous state with `dropdb notification_api` then recreate with `createdb notification_api` and restore your data with `pg_restore -Ft -d notification_api < ~/Downloads/my_local_db_backup.tar`; Then apply the `temp_db_setup_fixtures.sql` you just updated (you may have to tackle quite a few errors), or open `psql` and just copy-paste the lines from the script that you need.
-9. You may need to also copy these fixtures onto the preview database so the functional tests you've written will also run successfully there, for example:
+4. Update the function apply_fixtures() in notifications-api/app/functional_tests_fixtures/__init__.py to make changes to the fixtures. Fixtures should be added using the API in an idempotent way.
```
Comment (robinjam) [inline docs/update-local-db-fixtures.md:11]:
```suggestion
4. Update the function apply_fixtures() in [notifications-api/app/functional_tests_fixtures/__init__.py](https://github.com/alphagov/notifications-api/blob/main/app/functional_tests_fixtures/__init__.py) to make changes to the fixtures. Fixtures should be added using the API in an idempotent way.
```

**Review by robinjam — APPROVED** [review]
LGTM, nice!

Couple of non-blocking suggestions, feel free to merge them or discard them.

**Review by robinjam — APPROVED** [review]

**Pull request merged**